client/src/styling.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export const Main = styled.section`
+  padding: 1em;
+  background: papayawhip;
+  max-width: 500px;
+  font-family: Arial, Helvetica, sans-serif;
+  flex-direction: row;
+`;
+
+// Component Containers
+
+export const NamePriceContainer = styled.section`
+  padding: 0.5em;
+  background: white;
+`;
+
+export const DescriptionContainer = styled.section`
+  padding: 0.5em;
+  background: white;
+  margin: 0px 0px 30px;
+`;
+
+export const ColorContainer = styled.section`
```

Comment:
Your containers can likely be combined into a single `Container` class - the styling you have written is nearly identical. You can pass props to set unique styling.

Reply:
Yes! I built this all one component at a time and now I'm seeing a pattern :)